packages/libs/wdk-client/src/Components/AttributeFilter/Histogram.jsx

```diff
@@ -24,7 +24,7 @@ import {
 } from '../../Components/AttributeFilter/AttributeFilterUtils';
 import Icon from '../../Components/Icon/IconAlt';
 import { CollapsibleSection, IconAlt } from '../../Components';
-import Tooltip from '../../Components/Overlays/Tooltip';
+import Tooltip from '@veupathdb/coreui';
```

Comment:
This errors out because it's not a default import, so it needs to be wrapped in `{ }`